.circleci/config.yml

```diff
@@ -675,6 +675,20 @@ jobs:
           name: Create automatic PR
           command: bundle exec fastlane automatic_bump github_rate_limit:10
 
+  loadshedder-integration-tests:
+    <<: *base-job
+    steps:
+      - checkout
+      - setup-git-credentials
+      - trust-github-key
+      - install-dependencies
+      - run:
+          name: V3 LoadShedder integration tests
+          command: bundle exec fastlane v3_loadshedder_integration_tests
+      - run:
+          name: Backend and LoadShedder integration tests
+          command: bundle exec fastlane backend_integration_tests
```

Comment:
I haven't tied these to slack output yet, I can do it in a follow-up PR. @tonidero have you already done it for android? what's the setup?